.github/workflows/sync-token-issuer.yaml

```diff
@@ -0,0 +1,39 @@
+name: Sync token issuer code
+on:
+  pull_request:
+  push:
+    branches:
+      - develop
```

Comment:
Should this be main since there is no develop for this repo?

Reply:
yep, fixed!